PhotoTaggingGramplet/po/uk-local.po

```diff
@@ -0,0 +1,192 @@
+# Ukrainian translation for PhotoTaggingGramplet
+# This file is distributed under the same license as the Gramps package.
+#
+# Illia Maier <illia@maier.page>, 2024.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-06-24 11:30-0500\n"
+"PO-Revision-Date: 2024-10-09 23:44+0300\n"
+"Last-Translator: Illia Maier <illia@maier.page>\n"
+"Language-Team: Ukrainian <https://hosted.weblate.org/projects/gramps-project/"
+"gramps/uk/>\n"
+"Language: uk\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
+"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
+
+#: PhotoTaggingGramplet/PhotoTaggingGramplet.gpr.py:28
+#: PhotoTaggingGramplet/PhotoTaggingGramplet.gpr.py:37
+msgid "Photo Tagging"
+msgstr "Позначення на фото"
+
+#: PhotoTaggingGramplet/PhotoTaggingGramplet.gpr.py:29
+msgid "Gramplet for tagging people in photos"
+msgstr "Грамплет для позначення людей на фотографіях"
+
+#: PhotoTaggingGramplet/PhotoTaggingGramplet.py:108
+msgid "https://www.gramps-project.org/wiki/index.php/Photo_Tagging_Gramplet"
+msgstr ""
```

Comment:
Please keep the URL, even if it not translated.

Reply:
Done